uavcan/internet/60.HandleIncomingUDPPacket.uavcan

```diff
@@ -0,0 +1,52 @@
+#
+# This message carries UDP packets sent from a remote host on the Internet or a LAN to a node on the
+# local UAVCAN bus. Please refer to the defintion of the message type OutgoingUDPPacket for a
+# general overview of the packet forwarding logic.
+#
+# This data type has been made a service type rather than a message type in order to make the transfers
+# addressable, allowing the nodes to employ hardware CAN acceptance filters for filtering out
+# forwarded datagrams that are not addressed to them. Additionally, requiring the destination
+# nodes to always respond upon reception of the forwarded datagram opens interesting opportunities
+# for future extensions of the forwarding protocol.
```

Comment:
The future extensions mentioned in the description would lie outside of the scope of UDP. For example, a local node, having received an expected response from a remote host, could instruct the modem to remove the corresponding masquerading entry by setting a flag in the response data structure. Although this specific use case is rather questionable, in indicates what kind of extensions I'm talking about.